dist/api/pam.js

```diff
@@ -20,7 +20,7 @@ function listAccessPoliciesForGroups({ realmName, groupIds }, { apiUrl, queenCli
         groupIds.forEach(groupId => query.append('group_ids', encodeURIComponent(groupId)));
         // send request
         const response = yield queenClient.authenticator.tsv1Fetch(`${apiUrl}/v1/identity/pam/policies?${query.toString()}`, { method: 'GET' });
-        const data = yield utils_1.validateRequestAsJSON(response);
+        const data = yield (0, utils_1.validateRequestAsJSON)(response);
```

Comment:
@pirtleshell are these 0's config changes for style or why are they showing up now? Just curios what could have caused it

Reply:
i have no clue. gonna devote some prof dev time to figure it out this sprint.